$NetBSD: patch-ab,v 1.1 2010/09/10 05:41:27 taca Exp $

Use rake command name from environment if it is avaiable.

--- Rakefile.orig	2010-08-27 06:25:19.000000000 +0000
+++ Rakefile
@@ -194,11 +194,13 @@ task :package => [:makemo]
 ############################################################
 desc 'Run all tests'
 task :test do
+   rake = ENV['RAKE']
+   rake = 'rake' if rake.nil?
    Dir.chdir("test") do
      if RUBY_PLATFORM =~ /win32/
        sh "rake.bat", "test"
      else
-       sh "rake", "test"
+       sh rake, "test"
      end
    end
 end
